dsrc/parser.lit

```diff
@@ -484,6 +489,8 @@ else if (matchAll(line, regex("^---.+"))) {
     curBlock.isCodeblock = true;
     curBlock.name = strip(line[3..$]);
 
```

Comment:
I think it would be good if `+=` and `:=` didn't require the `---`, so as not to break backwards compatibility and also because it is convenient. This is a simple fix though, you could add this
```d
    if (curBlock.name.endsWith("+=")) {
        curBlock.modifiers ~= Modifier.additive;
        curBlock.name = strip(curBlock.name[0..$-2]);
    } else if (curBlock.name.endsWith(":=")) {
        curBlock.modifiers ~= Modifier.redef;
        curBlock.name = strip(curBlock.name[0..$-2]);
    }
```
right here to grab the `+=` or `:=` as a modifier.